### DISCUSSION TITLE
Use .gz or .gz.csi in jbrowse 1

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3495] [Q&A] [answered]
Hello,
I know that zipped files can be used in jbrowse 2 but they do not seem to be working on jbrowse 1. Is there a way so use .gz/.gz.csi/.gz.tbi files instead of unzipping them (because the unzipped files take too much storage)

Thanks

**Comment by cmdcolin** [comment, marked as answer]
what file type are you using? in JBrowse 1, BEDTabix, VCFTabix, and GFF3Tabix store classes can these files.

See https://jbrowse.org/docs/variants.html#variant-tracks-vcf for an example using VCF. It automatically infers the location of the .tbi file if it is the same as the vcf.gz with .tbi added.

For CSI, you can use the csiUrlTemplate config option

**Reply by cmdcolin** [reply]
note also that jbrowse 1 issues can be posted here https://github.com/GMOD/jbrowse/discussions not a big deal, just a note

**Reply by MehdiiYac** [reply]
> what file type are you using? in JBrowse 1, BEDTabix, VCFTabix, and GFF3Tabix store classes can these files.
> 
> See https://jbrowse.org/docs/variants.html#variant-tracks-vcf for an example using VCF. It automatically infers the location of the .tbi file if it is the same as the vcf.gz with .tbi added.
> 
> For CSI, you can use the csiUrlTemplate config option
Thanks, and what about the refsequence file, is it also possible to have it zipped?

**Reply by MehdiiYac** [reply]
> note also that jbrowse 1 issues can be posted here https://github.com/GMOD/jbrowse/discussions not a big deal, just a note

Oh sorry about that i didn't know that there was another repo for jbrowse 1, i will ask future questions there.
Thanks

**Reply by cmdcolin** [reply]
>Thanks, and what about the refsequence file, is it also possible to have it zipped?

The refsequence can use two different compressed formats, 2bit (.2bit files) and bgzip indexed fasta

Example config with .2bit in tracks.conf format

perform conversion to 2bit
```
faToTwoBit genome.fa genome.2bit
```

put into tracks.conf
```
[GENERAL]
refSeqs=genome.2bit
[tracks.refseqs]
urlTemplate=genome.2bit
```

Example config with BgzipIndexedFasta


prepare bgzip indexed fasta

```
bgzip -i genome.fa
samtools faidx genome.fa.gz
```

setup tracks.conf

```
[GENERAL]
refSeqs=genome.fa.gz.fai
[tracks.refseqs]
urlTemplate=genome.fa.gz
```
